sidekiq_with_redis_test.rb

```diff
@@ -0,0 +1,72 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require 'bundler/inline'
+
```

Comment:
Is this test in the root directory of the agent instead of in test/?

Reply:
Odd. Thanks for catching that, will address!

Reply:
Moved with [eb8f2ef](https://github.com/newrelic/newrelic-ruby-agent/pull/1640/commits/eb8f2efb850c7a30935945e6b8cedd71768d6e4a), thanks!